lib/ui/settings/data/import/qr_scanner_overlay.dart

```diff
@@ -0,0 +1,145 @@
+
```

Comment:
If you run the formatter the redundant empty lines should go away. You could configure your IDE to re-format automatically on save.

Reply:
Done